src/services/crmanagers/PushVariantResourceManager.js

```diff
@@ -64,7 +64,9 @@ export class PushVariantResourceManager extends GenericResourceManager {
   }
 
   list(user, res, labels) {
-    return Promise.all(res.variants.map(variantRes => super.list(user, variantRes, labels))).then(results => {
+    return Promise.all(
+      res.variants.map(variantRes => super.list(user, { ...variantRes, namespace: 'mobile-developer-console' }, labels))
```

Comment:
the namespace value should not be hardcoded. There is a function to return the namespace. See https://github.com/aerogear/mobile-developer-console/blob/master/src/services/openshift.js#L3

Reply:
Actuall, the res might already have the namespace field. It should be used for the variants as well.

Reply:
That was a mistake, left it in by accident when I was having PC issues! Will change now.

Reply:
> Actuall, the res might already have the namespace field. It should be used for the variants as well.

Yes I prefer that way. I am already doing it [here](https://github.com/aerogear/mobile-developer-console/pull/349/files#diff-80838ee30b66a4939e048729a9eee66bR87).